SimTracker/TrackAssociatorProducers/python/quickTrackAssociatorByHits_cfi.py

```diff
@@ -20,3 +20,6 @@
     quickTrackAssociatorByHits.associateStrip = False
     quickTrackAssociatorByHits.associatePixel = False
     quickTrackAssociatorByHits.useClusterTPAssociation = False
+
+eras.phase2_muon.toModify( quickTrackAssociatorByHits, pixelSimLinkSrc = cms.InputTag("simSiPixelDigis","Pixel") )
+eras.phase2_muon.toModify( quickTrackAssociatorByHits, stripSimLinkSrc = cms.InputTag("simSiPixelDigis","Tracker") )
```

Comment:
Yes, you are right it is not needed, but in the beginning for some reason it was complaining and I added those lines also in quickTrackAssociatorByHits_cfi.py. I reverted the file to its original version. or do you want to keep it?